components/klaudsolcms/modals/modal_body/EditCollectionTypeBody.js

```diff
@@ -0,0 +1,79 @@
+import { useState, useContext } from "react";
+import { Formik, Form, Field } from "formik";
+import { useRouter } from "next/router";
+import { loadEntityTypes } from "@/components/reducers/actions";
+import RootContext from "@/components/contexts/RootContext";
+
+export default function EditCollectionTypeBody({ formRef, hideModal }) {
+  const { state: rootState, dispatch: rootDispatch } = useContext(RootContext);
+  const router = useRouter();
+  const slug = router.query.entity_type_slug;
+
+  const entityTypes = rootState.entityTypes;
+  const currentEntityType = entityTypes.find(
+    (eType) => eType.entity_type_slug === slug
+  );
+  const name = currentEntityType?.entity_type_name;
+
+  const formikParams = {
+    initialValues: {
+      name,
+      slug,
+    },
+    innerRef: formRef,
+    onSubmit: (values) => {
+      console.error(JSON.stringify(values));
```

Comment:
Let's clean up debugging codes so that it won't pollute the users' console screen.

```suggestion
```